docs/source/dashboard.rst

```diff
@@ -0,0 +1,250 @@
+estimates it will take 10s to execute all the tasks it has currently been assigned.
+
+.. _dashboard.task-stream:
+
+Task Stream
+-----------
+
+The task stream is a view of all the tasks across worker-threads. Each row represents a thread and each rectangle represents 
+an individual tasks. The color for each rectangle corresponds to the task-prefix of the task being performed and it matches the color 
+of the *Progress* plot (see Progress section). This means that all the individual tasks part of the `inc` task-prefix for example, will have 
+the same randomly assigned color from the viridis color map. 
+
+There are certain colors that are reserved for a specific kinds of tasks:
+
+.. raw:: html
+
+    <table>
+        <tr>
+            <td><b>Color</b></td><td><b>Meaning</b></td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="light red square" style="color:rgba(255, 0, 0, 0.4); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Transferring data between workers tasks.</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="light orange square" style="color: rgba(255,165,0, 0.4); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Reading from or writing to disk.</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="light grey square" style="color:rgba(128,128,128, 0.4); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Serializing/deserializing data.</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="black square" style="color:rgba(0, 0, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Erred tasks.</td>
+        </tr>
+    </table>
+
+
+.. figure:: images/dashboard_taskstream_healthy.png
+    :alt: Task Stream pane showing eight horizontal bars, one per worker-thread, with different shades of 
+    greens and blues and some red. 
+
+.. figure:: images/dashboard_task_stream_unhealthy.png
+    :alt:  Task Stream pane showing twelve horizontal bars, one per worker-thread, with majority of white space
+    in each bar, and displaying some purple, red and orange rectangles.   
+
+
+In some scenarios the dashboard will have white spaces between each rectangle, this means that during that time the worker-thread
+is idle. Having too much white and red is an indication of not optimal use of resources.
+
+.. _dashboard.progress:
+
+Progress
+--------
+
+.. figure:: images/dashboard_progress.png
+    :alt: Progress pane showing horizontal bars for each task-prefix matching with the names "add", "double", "inc",
+    and "sum". The "double", "inc" and "add" bars have a progress of approximately one third of the total tasks, displayed 
+    in their individual color with different transparency levels. The "double" and "inc" bars have a grey background, and  
```

Comment:
```suggestion
    as individual colors that correspond with the task stream. The "double" and "inc" bars have a grey background, and  
```

Reply:
I wonder if we should mention level of transparency or not.